scripts/dummy.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
This is just the dummy script from the feedhenry/load-runner repo, re-written with promises.